core/src/test/java/dev/morphia/test/mapping/TestReferenceAnnotationPosition.java

```diff
@@ -0,0 +1,136 @@
+package dev.morphia.test.mapping;
+
+import static java.lang.annotation.ElementType.*;
+import static java.lang.annotation.RetentionPolicy.*;
+import static org.junit.Assert.*;
```

Comment:
core tests use `testng` and `junit` isn't on the classpath.  looks like you need a rebase as I recently excluded junit in core because it was leaking in due to transitive deps.

Reply:
Of course.